src/main/java/ldbc/snb/datagen/util/DateUtils.java

```diff
@@ -55,32 +61,22 @@ Linked Data Benchmark Council (http://www.ldbcouncil.org)
     private long simulationEnd;
     private long fromBirthDay;
     private long toBirthDay;
-    private GregorianCalendar calendar;
     private long bulkLoadThreshold;
     private PowerDistribution powerDist;
     private DateFormatter dateFormatter;
 
     // This constructor is for the case of friendship's created date generator
-    public DateUtils(LdbcConfiguration conf, GregorianCalendar simulationStartYear, GregorianCalendar simulationEndYear,
+    public DateUtils(LdbcConfiguration conf, LocalDate simulationStartYear, LocalDate simulationEndYear,
                      double alpha) {
-        simulationEndYear.setTimeZone(TimeZone.getTimeZone("GMT"));
-        simulationStartYear.setTimeZone(TimeZone.getTimeZone("GMT"));
-        simulationStart = simulationStartYear.getTimeInMillis();
-        simulationEnd = simulationEndYear.getTimeInMillis();
+        simulationStart = toEpochMilli(simulationStartYear);
+        simulationEnd = toEpochMilli(simulationEndYear);
         powerDist = new PowerDistribution(0.0, 1.0, alpha);
 
         // For birthday from 1980 to 1990
-        GregorianCalendar fromBirthCalendar = new GregorianCalendar(1980, 1, 1);
```

Comment:
So provided that months in `GregorianCalendar` are [0-based], itt seems we never had people born in Jan 1980. And consequently we had less 'large posters' (as people born in January are the large posters according to `isLargePoster`).

Reply:
And it seems I fixed this bug by mistake.